src/test/java/com/fincatto/nfe310/classes/danfe/NFGeraDanfeTest.java

```diff
@@ -18,17 +18,19 @@
 	public void deveGerarDanfeAPartirDoXML() throws IOException {
 		final NFNota nota = new NFNota();
 		final NFSignature assinatura = new NFSignature();
-		assinatura.setSignatureValue("NFe89172658591754401086218048846976493475937081");
+		assinatura
+				.setSignatureValue("NFe89172658591754401086218048846976493475937081");
 		final NFNotaInfo notaInfo = FabricaDeObjetosFake.getNFNotaInfo();
 		final int identificadorLocal = 123456;
 		nota.setAssinatura(assinatura);
 		nota.setInfo(notaInfo);
 		nota.setIdentificadorLocal(identificadorLocal);
 		nota.toString();
-		FileOutputStream fos = new FileOutputStream("/tmp/danfe.pdf");
-		fos.write(NFDanfeReport.imprimirDanfe(nota));
-		fos.close();
-		Assert.assertTrue(new File("/tmp/danfe.pdf").exists());
+		try (FileOutputStream fos = new FileOutputStream(System.getProperty("java.io.tmpdir")+"/"+"danfe.pdf")) {
+			fos.write(NFDanfeReport.imprimirDanfe(nota));
+			fos.close();
```

Comment:
Ajustado @fincatto!

2016-06-13 13:52 GMT-03:00 Diego Fincatto <notifications@github.com>:

> In src/test/java/com/fincatto/nfe310/classes/danfe/NFGeraDanfeTest.java
> <https://github.com/wmixvideo/nfe/pull/163#discussion_r66827239>:
>
> >  		final NFNotaInfo notaInfo = FabricaDeObjetosFake.getNFNotaInfo();
> >  		final int identificadorLocal = 123456;
> >  		nota.setAssinatura(assinatura);
> >  		nota.setInfo(notaInfo);
> >  		nota.setIdentificadorLocal(identificadorLocal);
> >  		nota.toString();
> > -		FileOutputStream fos = new FileOutputStream("/tmp/danfe.pdf");
> > -		fos.write(NFDanfeReport.imprimirDanfe(nota));
> > -		fos.close();
> > -		Assert.assertTrue(new File("/tmp/danfe.pdf").exists());
> > +		try (FileOutputStream fos = new FileOutputStream(System.getProperty("java.io.tmpdir")+"/"+"danfe.pdf")) {
> > +			fos.write(NFDanfeReport.imprimirDanfe(nota));
> > +			fos.close();
>
> Neste caso, o close não mais faz-se necessário.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/wmixvideo/nfe/pull/163/files/3c3ad367d1fa17c71a72d9ef75bd866eca651eaf#r66827239>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AE95njxHglJ1St6Gb784Tt1XjCx3uQfqks5qLYrfgaJpZM4I0g7s>
> .
>



-- 
Felipe Wender Mendonça Martins
#Github: https://github.com/felipewmartins
#blog: http://felipewmartins.github.io/